docs/build/tutorials/scaffold-eth.md

```diff
@@ -116,15 +119,15 @@ First, you can compile your contract by running:
 ```bash
 yarn compile
 ```
-![Compile](/img/build/tutorials/sc-compile.png)
+![Compile](/img/build/s2.png)
 
 Then, you can run the following command from the root directory of your project:
 
 
 ```
 yarn deploy
 ```
-![Deploy](/img/build/tutorials/sc-deploy.png)
+![Deploy](/img/build/s6.png)
```

Comment:
the s6 image seems to be the image of compilation and not for deployment. Kindly take a look. See the deployment guide [here](https://docs.kaia.io/build/tutorials/scaffold-eth/#setting-up-development-environment-)